lib/vrl/web-playground/public/index.html

```diff
@@ -88,83 +88,150 @@ <h1>[draft] VRL playground wasm</h1>
+            window.eventEditor.setValue(JSON.stringify(urlState["event"], null, "\t"));
+
+            console.log("[DEBUG::queryStringLogic] Current Params:", JSON.parse(atob(stateParam)));
+            let res = handleRunCode(JSON.parse(atob(stateParam)));
+            console.log("[DEBUG::queryStringLogic] Running VRL with current Params:", res);
+          } catch (e) {
+            window.outputEditor.setValue(`Error reading the shared URL\n${e}`);
+          }
+        }
       });
+    </script>
 
-      window.outputEditor = monaco.editor.create(document.getElementById('container-output'), {
-        language: 'json',
-        theme: 'vs-light',
-        minimap: { enabled: false }
+    <script type="module">
+      import init, { run_vrl } from "./vrl_web_playground.js";
+      init().then(() => {
+        window.run_vrl = run_vrl;
       });
-      const queryString = window.location.search;
-      if (queryString.length != 0) {
-        const urlParams = new URLSearchParams(queryString);
-        const stateParam = urlParams.get('state');
-
-        let urlState = JSON.parse(atob(stateParam));
-
-        window.programEditor.setValue(urlState["program"]);
-        window.eventEditor.setValue(JSON.stringify(urlState["event"], null, '\t'));
-
-        console.log("[DEBUG::queryStringLogic] Current Params:", JSON.parse(atob(stateParam)));
-        let res = handleRunCode(JSON.parse(atob(stateParam)));
-        console.log("[DEBUG::queryStringLogic] Running VRL with current Params:", res);
+    </script>
+
+    <script>
+      function tryJsonParse(str) {
+        try {
+          return JSON.parse(str);
+        } catch (e) {
+          monaco.languages.json.jsonDefaults.setDiagnosticsOptions({
+            validate: false
+          });
+          window.outputEditor.setValue(`Error attempting to parse the following string into valid JSON\n
+String: ${str}
+\nEnsure that the Event editor contains valid JSON
+\nCommon mistakes:\n
+  Trailing Commas\n  Last line is a newline or whitespace\n  Unbalanced curly braces
+  If using JSONL, ensure each line is valid JSON`);
+        }
       }
-    });
-  </script>
-
-  <script type="module">
-    import init, { run_vrl } from "./vrl_web_playground.js";
-    init().then(() => {
-      window.run_vrl = run_vrl;
-      // console.log(run_vrl({"program": ".", "event": {}}));
-    });
-  </script>
-
-  <script>
-    function handleRunCode(input) {
-      if (input == null) {
-        input = {
-          program: window.programEditor.getValue(),
-          event: JSON.parse(window.eventEditor.getValue())
+
+      function isJsonL() {
+        if (window.eventEditor.getModel().getLineCount() > 1) {
+          let lines = window.eventEditor.getModel().getLinesContent();
+          // if the second line is a json object
+          // we assume the user has passed in valid json on each
+          // line
+          if (lines[1][0] == "{" && lines[1][lines[1].length - 1] == "}") {
```

Comment:
this is a fairly simple assumption
"if the second line of the event editor is of the format "{ somethings in side here }" then we assume the user wanted to pass in JSONL

the program later attempts to use `run_vrl()` on each of these events